en/docs/setup/deployment-guidelines-in-production.md

```diff
@@ -281,18 +247,15 @@ you can take the following approaches:
     sure to store sensitive data such as username and password to
     connect to the registry in a property file instead of in the Java
     code and secure the properties file with the [secure
-    vault](../../admin-guide/carbon-secure-vault-implementation)
+    vault](https://docs.wso2.com/display/ADMIN44x/Carbon+Secure+Vault+Implementation)
```

Comment:
Aren't we supposed to give the relative path here?

Reply:
Fixed